consumer/init_consumer.py

```diff
@@ -1,18 +1,21 @@
-import sys, os
+import os
 #import boto3
 
 from consumer import Consumer
 from kafka_message.ttypes import KafkaMessage
 from thrift.TSerialization import deserialize
 from cassandra.cluster import Cluster
 
+
+KAFKA_HOST = os.environ['KAFKA_HOST']
+CASSANDRA_HOST = os.environ['CASSANDRA_HOST']
```

Comment:
These constants should be defined only when script is running as `__main__` IMO i.e. within `run()`